Assets/AppCenter/Plugins/AppCenterSDK/Distribute/Shared/UpdateAction.cs

```diff
@@ -1,21 +1,35 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT license.
-
-namespace Microsoft.AppCenter.Unity.Distribute
-{
-    /// <summary>
-    /// User update action.
-    /// </summary>
-    public enum UpdateAction
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+namespace Microsoft.AppCenter.Unity.Distribute
+{
+    /// <summary>
+    /// User update action.
+    /// </summary>
+    public enum UpdateAction
     {
-        /// <summary>
-        /// Action to trigger the download of the release.
-        /// </summary>
-        Update,
+#if UNITY_ANDROID

```

Comment:
It fixes the SDK. Please update changelog

Reply:
What does it fix?

Reply:
@jaelim-ms it changes constants for android (Did it work before?)

Well, actually I would keep this public part as is and fix the Android binding if required